### PR TITLE
[ODS-4237] Ed-Fi-Alliance repo paths in nuspec files 

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: EdFi.Ods.Utilities.Migration.nuspec [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <version>$PackageVersion$</version>
     <authors>Ed-Fi Alliance</authors>
     <owners>Ed-Fi Alliance</owners>
+	<projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-MigrationUtility</projectUrl>
```
Comment (jamessmckay) [inline EdFi.Ods.Utilities.Migration.nuspec]: fix formatting

File: EdFi.Ods.Utilities.Migration/EdFi.Ods.Utilities.Migration.csproj [hunk 2]
```diff
@@ -16,8 +16,8 @@
     <Description>Database migration utility for the Ed-Fi ODS/API solution.</Description>
     <Copyright>Copyright © 2020 Ed-Fi Alliance, LLC and contributors</Copyright>
     <PackageId>EdFi.Ods.Utilities.Migration</PackageId>
-    <PackageProjectUrl>https://github.com/Ed-Fi-Alliance/Ed-Fi-MigrationUtility/tree/master</PackageProjectUrl>
-    <License>https://www.ed-fi.org/license</License>
+    <PackageProjectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-MigrationUtility/tree/master</PackageProjectUrl>
```
Comment (jamessmckay) [inline EdFi.Ods.Utilities.Migration/EdFi.Ods.Utilities.Migration.csproj:19]: fix formatting

File: EdFi.Ods.Utilities.Migration/EdFi.Ods.Utilities.Migration.csproj [hunk 3]
```diff
@@ -26,7 +26,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <DebugType>full</DebugType>
     <DebugSymbols>true</DebugSymbols>
-  </PropertyGroup>
+  </PropertyGroup>
```
Comment (jamessmckay) [inline EdFi.Ods.Utilities.Migration/EdFi.Ods.Utilities.Migration.csproj]: remove extra white space

File: EdFi.Ods.Utilities.Migration/EdFi.Ods.Utilities.Migration.csproj [hunk 4]
```diff
@@ -43,5 +43,6 @@
     <None Include="Scripts/**" CopyToOutputDirectory="Always" />
     <None Include="Sample Calendar Config/**" CopyToOutputDirectory="Always" />
     <None Update="log4net.config" CopyToOutputDirectory="Always" />
+	<None Include="../LICENSE.txt" Pack="true" PackagePath="LICENSE.txt"/>
```
Comment (jamessmckay) [inline EdFi.Ods.Utilities.Migration/EdFi.Ods.Utilities.Migration.csproj]: fix formatting

**Review by jamessmckay — CHANGES_REQUESTED** [review]

File: EdFi.Ods.Utilities.Migration.nuspec (hunk 1 above)
Comment (semalaiappan) [reply EdFi.Ods.Utilities.Migration.nuspec]: I fixed it . Thanks

File: EdFi.Ods.Utilities.Migration/EdFi.Ods.Utilities.Migration.csproj (hunk 3 above)
Comment (semalaiappan) [reply EdFi.Ods.Utilities.Migration/EdFi.Ods.Utilities.Migration.csproj]: I fixed it . Thanks

File: EdFi.Ods.Utilities.Migration/EdFi.Ods.Utilities.Migration.csproj (hunk 4 above)
Comment (semalaiappan) [reply EdFi.Ods.Utilities.Migration/EdFi.Ods.Utilities.Migration.csproj]: I fixed it . Thanks

**Comment by semalaiappan** [comment]
Fixed formatting on this PR . Thanks

File: EdFi.Ods.Utilities.Migration/EdFi.Ods.Utilities.Migration.csproj (hunk 2 above)
Comment (semalaiappan) [reply EdFi.Ods.Utilities.Migration/EdFi.Ods.Utilities.Migration.csproj:19]: fixed it . Thanks

Comment (semalaiappan) [reply EdFi.Ods.Utilities.Migration/EdFi.Ods.Utilities.Migration.csproj:19]: Fixed it . Thanks

**Review by jamessmckay — APPROVED** [review]

**Pull request merged**